user-manual/import-export/export-xml.rst

```diff
@@ -464,3 +464,24 @@ will allow you to "unzip" a ZIP file.
   :alt: An image of the jobs page after an XML export has been executed
 
 :ref:`Back to top <export-xml>`
+
+**To include digital objects with your XML export:**
+
+Some AtoM sites are configured to enable users to include attached digital
```

Comment:
Same comments - maybe a link to the settings page; maybe a clarification on the tradeoff of exporting digital objects vs hierarchies?

Reply:
Agreed, make "configured" a link to https://www.accesstomemory.org/en/docs/2.7/user-manual/administer/settings/#clipboard-settings. And add to line 484 "This means you will need to individually add the descriptions with digital objects to the clipboard if you want to export them, rather than just adding a parent description."